### ISSUE TITLE
Missmatch between documentation and actual packages. readme.md not updated?

### ISSUE THREAD

**Issue by biels** [react-keycloak/react-keycloak#94]
Packages are core, nextjs, razzle, ssr and web. However, documentation talks about a native package and does not mention many others. Am I missing something or is the readme just not updated?

**Comment by panz3r** [comment]
Hi @biels ,

Thanks for your feedback, actually there's nothing wrong about the `README` file.

The `nextjs` and `razzle` packages have been deprecated in favour of the `ssr` one and thus have been removed from the `README` (but for now are still available inside the `packages` folder - they'll get removed with the next major version, see `next` branch).

The `native` package (short for `react-native`) is not included in this repository but is available at [`react-keycloak/react-native-keycloak`](https://github.com/react-keycloak/react-native-keycloak), this mainly because it was developed from another developer and then moved to the `react-keycloak` organisation.

In short the available and supported packages are:

- web (listed as `React`)
- ssr
- native (listed as `React Native`)

all of which are listed in the main `README` file ([here](https://github.com/react-keycloak/react-keycloak/blob/master/README.md)).

**Comment by biels** [comment]
Thanks for your reply. I have tried using ssr on a nextjs app without success. I opened an issue on https://github.com/react-keycloak/react-keycloak-examples/issues/5 to update the nextjs example. I can help with whatever is necessary.
